receiver/sapnetweaverreceiver/metrics.go

```diff
@@ -280,6 +280,61 @@ func (s *sapNetweaverScraper) recordSapnetweaverWorkProcessActiveCountDataPoint(
 	s.mb.RecordSapnetweaverWorkProcessActiveCountDataPoint(now, int64(similarityCount), systemWPTableResponse.Workprocess.Item[len(systemWPTableResponse.Workprocess.Item)-1].Instance, systemWPTableResponse.Workprocess.Item[len(systemWPTableResponse.Workprocess.Item)-1].Typ, systemWPTableResponse.Workprocess.Item[len(systemWPTableResponse.Workprocess.Item)-1].Status)
 }
 
+// recordSapnetweaverAbapRfcCountDataPoint
+func (s *sapNetweaverScraper) recordSapnetweaverAbapRfcCountDataPoint(now pcommon.Timestamp, rfcTable string) {
+	rfcConnectionTable := parseRfcConnectionsTable(rfcTable)
+	for rfcType, count := range rfcConnectionTable {
+		s.mb.RecordSapnetweaverAbapRfcCountDataPoint(now, count, rfcType)
+	}
+}
+
+func parseRfcConnectionsTable(rfcTable string) map[string]int64 {
+	rfcTypes := map[string]int64{}
+	if strings.Contains(rfcTable, "Communication Table is empty") {
+		return rfcTypes
+	}
+
+	for _, line := range strings.Split(rfcTable, "\n") {
+		// select content between | and |, but not the header with  --- header fields, i.e. type
+		if strings.Contains(line, "|") && !strings.Contains(line, "-") && !strings.Contains(line, "type") {
```

Comment:
In case it's legal to use a "-" in a name, we should look for headers by looking for at least three in a row "---"

Reply:
updated.